php_phongo.c

```diff
@@ -979,15 +986,14 @@ void php_phongo_read_preference_to_zval(zval *retval, const mongoc_read_prefs_t
 #else
 		ADD_ASSOC_ZVAL_EX(retval, "tags", state.zchild);
 #endif
-	} else {
-		ADD_ASSOC_NULL_EX(retval, "tags");
 	}
 } /* }}} */
 
 void php_phongo_write_concern_to_zval(zval *retval, const mongoc_write_concern_t *write_concern) /* {{{ */
 {
 	const char *wtag = mongoc_write_concern_get_wtag(write_concern);
 	const int32_t w = mongoc_write_concern_get_w(write_concern);
+	const int32_t wtimeout = mongoc_write_concern_get_wtimeout(write_concern);
```

Comment:
Does this count as mixing code and declarations? I actually would have thought the compiler would yell about the existing `*wtag` and `w` lines.